lithops/serverless/backends/k8s/k8s.py

```diff
@@ -223,7 +247,7 @@ def deploy_runtime(self, docker_image_name, memory, timeout):
 
         logger.info(f"Deploying runtime: {docker_image_name} - Memory: {memory} Timeout: {timeout}")
         self._create_container_registry_secret()
-        runtime_meta = self._generate_runtime_meta(docker_image_name)
+        runtime_meta = self._generate_runtime_meta(docker_image_name, memory)
```

Comment:
What is the reason of this change? is this necessary for you? can you revert it if it is not necessary?